docs/markdown-examples.md

```
@@ -0,0 +1,93 @@
+
+## More
+
+Check out the documentation for the [full list of markdown extensions](https://vitepress.dev/guide/markdown).
+
+
+##
+
+```mermaid
+flowchart
+    test --> test2
+```
```

Markdownlint (0.35.0)

Files should end with a single newline character

(MD047, single-trailing-newline)